internal/commands/result.go

```diff
@@ -1362,15 +1371,17 @@ func ReadResults(
 	resultsWrapper wrappers.ResultsWrapper,
 	exportWrapper wrappers.ExportWrapper,
 	scan *wrappers.ScanResponseModel,
-	params map[string]string,
+	resultsParams map[string]string,
 ) (results *wrappers.ScanResultsCollection, err error) {
 	var resultsModel *wrappers.ScanResultsCollection
 	var errorModel *wrappers.WebError
 
-	params[commonParams.ScanIDQueryParam] = scan.ID
-	_, sastRedundancy := params[commonParams.SastRedundancyFlag]
+	resultsParams[commonParams.ScanIDQueryParam] = scan.ID
+	_, sastRedundancy := resultsParams[commonParams.SastRedundancyFlag]
+
+	scaHideDevAndTestDep := resultsParams[ScaExcludeResultTypesParam] == ScaDevAndTestExclusionParam
 
-	resultsModel, errorModel, err = resultsWrapper.GetAllResultsByScanID(params)
+	resultsModel, errorModel, err = resultsWrapper.GetAllResultsByScanID(resultsParams)
```

Comment:
dont you need to add the changes to the HTTP client as well?

Reply:
no, this resultsParams will be recieved as a parameter in the HTTP client request funtion